Index: src/main/java/gov/loc/repository/bagit/driver/CommandLineBagDriver.java
===================================================================
--- src/main/java/gov/loc/repository/bagit/driver/CommandLineBagDriver.java	(revision 2137)
+++ src/main/java/gov/loc/repository/bagit/driver/CommandLineBagDriver.java	(revision 2138)
@@ -516,7 +516,6 @@
 				if (! result.isSuccess()) {
 					ret = RETURN_FAILURE;
 				}
-				return RETURN_SUCCESS;
 			} else if (OPERATION_VERIFYCOMPLETE.equals(operation.name)) {				
 				CompleteVerifier completeVerifier = new CompleteVerifierImpl();
 				completeVerifier.setMissingBagItTolerant(config.getBoolean(PARAM_MISSING_BAGIT_TOLERANT, false));				
